retroachievements/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+
+RetroAchievements API Wrapper
+---
+
+Official Python API wrapper for RetroAchievements
+
+"""
+
+__title__ = "retroachievements-api"
+__authors__ = "drisc"
+__version__ = "0.1"
```

Comment:
semver may be preferable.

Reply:
Can this be referenced in the user agent directly? I assume this makes it easier to maintain the version number.

Reply:
Was just about to commit that update haha

Reply:
The user-agent now references the __version__ in __init__.py

Reply:
That is a good change. It's a separate issue from the original comment, though. It may be preferable for this package to use semantic versioning.

Reply:
I updated it to use semver in the same commit: https://github.com/RetroAchievements/api-python/pull/1/files#diff-40d00a57b07939167fe7ec4058a6d4cee495453a268e1c9bd7123857b4b92525R12

Reply:
Ah, my mistake. I did not see it. Thank you 🙏